esp-wifi/Cargo.toml

```diff
@@ -74,3 +84,5 @@ wifi = []
 ble = [ "esp32-hal?/bluetooth" ]
 phy_enable_usb = []
 ps_min_modem = []
+esp_now = [ "wifi", "embedded-svc" ]
```

Comment:
Should we unify the feature name style? I.e decide on either `_` or `-`, currently we mixed between the two and I keep forgetting :D